diff -uNr subscription-manager-1.9.11/etc-conf/rhsm.conf subscription-manager-1.9.11-new/etc-conf/rhsm.conf
--- subscription-manager-1.9.11/etc-conf/rhsm.conf	2013-10-17 04:41:00.000000000 +0800
+++ subscription-manager-1.9.11-new/etc-conf/rhsm.conf	2014-08-20 03:54:49.702994000 +0800
@@ -3,13 +3,13 @@
 # Unified Entitlement Platform Configuration
 [server]
 # Server hostname:
-hostname = subscription.rhn.redhat.com
+hostname = candlepin.eayun.com
 
 # Server prefix:
-prefix = /subscription
+prefix = /candlepin
 
 # Server port:
-port = 443
+port = 8443
 
 # Set to 1 to disable certificate validation:
 insecure = 0
@@ -32,13 +32,13 @@
 
 [rhsm]
 # Content base URL:
-baseurl= https://cdn.redhat.com
+baseurl= https://pulp.zhaochao.eayunos.eayun.com/pulp/repos
 
 # Server CA certificate location:
 ca_cert_dir = /etc/rhsm/ca/
 
 # Default CA cert to use when generating yum repo configs:
-repo_ca_cert = %(ca_cert_dir)sredhat-uep.pem
+repo_ca_cert = %(ca_cert_dir)scandlepin-ca.pem
 
 # Where the certificates should be stored
 productCertDir = /etc/pki/product
diff -uNr subscription-manager-1.9.11/src/subscription_manager/repolib.py subscription-manager-1.9.11-new/src/subscription_manager/repolib.py
--- subscription-manager-1.9.11/src/subscription_manager/repolib.py	2013-10-17 04:41:00.000000000 +0800
+++ subscription-manager-1.9.11-new/src/subscription_manager/repolib.py	2014-08-20 04:12:47.346996587 +0800
@@ -286,7 +286,7 @@
         ('enabled', 1, '1'),
         ('gpgcheck', 1, '1'),
         ('gpgkey', 0, None),
-        ('sslverify', 1, '1'),
+        ('sslverify', 1, '0'),
         ('sslcacert', 0, None),
         ('sslclientkey', 0, None),
         ('sslclientcert', 0, None),
Binary files subscription-manager-1.9.11/src/subscription_manager/.repolib.py.swp and subscription-manager-1.9.11-new/src/subscription_manager/.repolib.py.swp differ
